R/utilities.R

```diff
@@ -1148,61 +1195,50 @@ assign_cn_to_ssm = function(this_sample,
-
+#'
 estimate_purity = function(in_maf,
                            in_seg,
                            sample_id,
-                           seg_file_source="ichorCNA",
-                           show_plots=FALSE,
-                           assume_diploid=FALSE,
-                           coding_only=FALSE,
-                           genes
-                           ){
+                           seg_file_source = "ichorCNA",
+                           show_plots = FALSE,
+                           assume_diploid = FALSE,
+                           coding_only = FALSE,
+                           genes){
 
   # Merge the CN info to the corresponding MAF file, uses GAMBLR function
   if(missing(in_maf) & missing(in_seg)){
-    CN_new <- assign_cn_to_ssm(this_sample = sample_id,coding_only=coding_only,assume_diploid=assume_diploid,genes=genes)$maf
-
+    CN_new = assign_cn_to_ssm(this_sample = sample_id, coding_only = coding_only, assume_diploid = assume_diploid, genes = genes)$maf
   }else if(!missing(in_seg)){
-    if(seg_file_source=="ichorCNA"){
-      CN_new <- assign_cn_to_ssm(maf_file = in_maf, seg_file = in_seg, seg_file_source = "ichorCNA",coding_only=coding_only,genes=genes)$maf
+    if(seg_file_source == "ichorCNA"){
+      CN_new = assign_cn_to_ssm(maf_file = in_maf, seg_file = in_seg, seg_file_source = "ichorCNA", coding_only = coding_only, genes = genes)$maf
     }else{
-      CN_new <- assign_cn_to_ssm(maf_file = in_maf, seg_file = in_seg, seg_file_source = "",coding_only=coding_only,genes=genes)$maf
+      CN_new = assign_cn_to_ssm(maf_file = in_maf, seg_file = in_seg, seg_file_source = "", coding_only = coding_only, genes = genes)$maf
     }
   }else{
     # If no seg file was provided and assume_diploid paramtere is set to true,
     if(assume_diploid){
-      CN_new <- assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE,coding_only=coding_only,genes=genes)$maf
+      CN_new = assign_cn_to_ssm(maf_file = in_maf, assume_diploid = TRUE, coding_only = coding_only, genes = genes)$maf
     }
   }
-
   # Change any homozygous deletions (CN = 0) to 1 for calculation purposes
   CN_new$CN[CN_new$CN<1] = 1
-
   # Select only the relevant columns, add new columns for VAF, Ploidy, and Purity
-  CN_new <- CN_new %>%
+  CN_new = CN_new %>%
     dplyr::select(Hugo_Symbol, Chromosome, Start_Position, End_Position, t_ref_count, t_alt_count, CN) %>%
-    dplyr::mutate(VAF = t_alt_count/(t_ref_count+t_alt_count)) %>%
+    dplyr::mutate(VAF = t_alt_count/(t_ref_count + t_alt_count)) %>%
     dplyr::mutate(Ploidy = ifelse(CN %in% 1:2, 1, 0)) %>%
     dplyr::mutate(Purity = "") %>%
     tidyr::drop_na(CN) %>%
-    tidyr::unite("Chrom_pos", Chromosome:Start_Position, sep=":", remove=FALSE)
+    tidyr::unite("Chrom_pos", Chromosome:Start_Position, sep = ":", remove = FALSE)
 
   # Create an empty list
   indiv_CN = list()
 
-  # For each copy number state, separate mutations in each CN state into separate datatables
```

Comment:
Kristena probably wants these comments left here

Reply:
Added back in.